store/storeapp/management/commands/create_client.py

```diff
@@ -0,0 +1,16 @@
+import random
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from store.storeapp.models import ClientModel
```

Comment:
Тут неверный импорт
```suggestion
from storeapp.models import ClientModel
```
и команда должна начинаться так `python manage.py create_client`

У меня ваш проект заработал. Уверен у вас тоже все получится